apprise/plugins/NotifyD7Networks.py

```diff
@@ -229,40 +197,41 @@ def send(self, body, title='', notify_type=NotifyType.INFO, **kwargs):
         # error tracking (used for function return)
         has_error = False
 
-        auth = '{user}:{password}'.format(
-            user=self.user, password=self.password)
-
-        # Python 3's versio of b64encode() expects a byte array and not
-        # a string.  To accommodate this, we encode the content here
-        auth = auth.encode('utf-8')
-
         # Prepare our headers
         headers = {
             'User-Agent': self.app_id,
+            'Content-Type': 'application/json',
             'Accept': 'application/json',
-            'Authorization': 'Basic {}'.format(base64.b64encode(auth))
+            'Authorization: Bearer ': self.token,
```

Comment:
There is a mistake here, should `'Authorization': f'Bearer {self.token}',`

Reply:
So if I test it:
```
2022-12-19 10:42:03,953 - WARNING - A Connection error occurred sending D7 Networks:xxxxxxx notification.
2022-12-19 10:42:03,953 - DEBUG - Socket Exception: Invalid leading whitespace, reserved character(s), or returncharacter(s) in header name: 'Authorization: Bearer '
```

Reply:
Great find on your part; i just pushed the change you pointed out  Do you still get the Error with the change you found?